ProcSDF/GUI/NodeEditor.cpp

```diff
@@ -10,6 +12,24 @@ void NodeEditor::draw()
 	{
 		NodeGraph::getSingleton()->recompileNodeGraph();
 	}
+	
+	ImGui::SameLine();
+
+	if (ImGui::Button("Save Project"))
+	{
+		ProjectSaver::saveProject();
+	}
+
+	ImGui::SameLine();
+
+	if (ImGui::Button("Load Project"))
+	{
+		bool isParseSucessful = ProjectSaver::loadProject();
+		if (!isParseSucessful) {
+			ERR("Error in parsing uploaded file");
+		}
+	}
+
```

Comment:
Move this to the title bar if possible? So, that we can have the normal `file -> load` and `file -> save/save as`  flow as expected in software. Check if this can be easily done or we'll just postpone this to later.